sycl/test-e2e/Matrix/joint_matrix_bfloat16_accumulator.cpp

```diff
@@ -0,0 +1,27 @@
+//==-------- joint_matrix_bfloat16.cpp  - DPC++ joint_matrix----------- ----==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// SG size = 32 is not currently supported for SYCL Joint Matrix by IGC on DG2
+// UNSUPPORTED: gpu-intel-dg2
```

Comment:
This should be added to the SG32/joint_matrix_bfloat16_accumulator.cpp
which you should also add